backend/src/ServiceLocator.ts

```diff
@@ -5,7 +5,11 @@ import ElectionRollDB from "./Models/ElectionRolls";
 import CastVoteStore from "./Models/CastVoteStore";
 import EmailService from "./Services/Email/EmailService";
 import { IBallotStore } from "./Models/IBallotStore";
+import { IEventQueue } from "./Services/EventQueue/IEventQueue";
+import PGBossEventQueue from "./Services/EventQueue/PGBossEventQueue";
+
 import AccountService from "./Services/Account/AccountService"
+import GlobalData from "./Services/GlobalData";
```

Comment:
I think Services/GlobalData is missing from the commit

Reply:
whoops, great catch, thanks!